.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## PR Requirements:
+ - [ ] The code must be reviewed by at least one Maintainer
+ - [ ] JIRA Ticket: https://jira.wh-sdlc.watson-health.ibm.com/browse/JIRA_TICKET_HERE
+ - [ ] 90 % test code coverage
+ - [ ] README updates
```

Comment:
I don't think you need to add this file to this branch. Only the `main` branch needs them.

Reply:
Deleted ✅